Source/Fuse.PushNotifications/Fuse.PushNotifications.unoproj

```diff
@@ -13,11 +13,9 @@
     "Fuse.Elements",
     "Fuse.Reactive",
     "Fuse.Scripting",
-    "Fuse.Common"
-  ],
-  "Projects": [
-    "../Experimental.TextureLoader/Experimental.TextureLoader.unoproj",
-    "../Fuse.Platform/Fuse.Platform.unoproj"
+    "Fuse.Common",
```

Comment:
I'm not sure how this change is relevant to the feature?

Reply:
I'm sorry, this is something I had to change to make it works in my PC, I forgot to add it before to pull.